test/jdk/javax/swing/JFileChooser/FileChooserSymLinkTest.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+/* @test
+   @bug 8281966
+   @key headful
+   @requires (os.family == "windows")
+   @library /java/awt/regtesthelpers
+   @build PassFailJFrame
+   @summary Test to check if the absolute path of Symbolic Link folder
+            is valid on ValueChanged property listener.
+   @run main FileChooserSymLinkTest
+*/
```

Comment:
May I ask to place these tags before the class declaration?

Add the asterisk to each line, like you see in the copyright and other tests.

Reply:
Updated.